e2e/tests/upgrades/upgrade_test.go

```diff
@@ -604,6 +606,35 @@ func (s *UpgradeTestSuite) TestV6ToV7ChainUpgrade() {
 	})
 }
 
+func (s *UpgradeTestSuite) TestV7ChainUpgradeAddLocalhost() {
+	t := s.T()
+	testCfg := testconfig.FromEnv()
+
+	ctx := context.Background()
+	_, _ = s.SetupChainsRelayerAndChannel(ctx)
+	chain, _ := s.GetChains()
+
+	s.Require().NoError(test.WaitForBlocks(ctx, 5, chain), "failed to wait for blocks")
+
+	t.Run("upgrade chain", func(t *testing.T) {
+		govProposalWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+		s.UpgradeChain(ctx, chain, govProposalWallet, v71.UpgradeName, testCfg.ChainAConfig.Tag, testCfg.UpgradeTag)
```

Comment:
this needs to be updated?

Reply:
updated! need to get the fix in #3172 included in this branch too. The `else if` we added earlier caused some weird issues in CI.